apps/chat/agent/tools.py

```diff
@@ -2,17 +2,24 @@
 import logging
 import uuid
 from datetime import datetime, timedelta
+from importlib import import_module
 
 from django_celery_beat.models import ClockedSchedule, IntervalSchedule, PeriodicTask
 from langchain.tools.base import BaseTool
 
 from apps.chat.agent import schemas
 from apps.events.models import ScheduledMessage
-from apps.experiments.models import ExperimentSession
+from apps.experiments.models import AgentTools, ExperimentSession
 from apps.utils.time import pretty_date
 
 BOT_MESSAGE_FOR_USER_TASK = "apps.chat.tasks.send_bot_message_to_users"
 
+TOOL_CLASS_MAP = {
+    AgentTools.SCHEDULE_UPDATE: "apps.chat.agent.tools.UpdateScheduledMessageTool",
+    AgentTools.ONE_OFF_REMINDER: "apps.chat.agent.tools.OneOffReminderTool",
+    AgentTools.RECURRING_REMINDER: "apps.chat.agent.tools.RecurringReminderTool",
+}
```

Comment:
if you put this below the class definitions you could reference the classes directly instead of as strings

Reply:
I did have it there at first, but it just felt like a weird place to put it. I can't really express the reason I'm against putting it there. Maybe I should just get over it. Ok, I'll update

Reply:
I know the feeling

Reply:
https://github.com/dimagi/open-chat-studio/pull/417/commits/75b76722b668f89955574e0008f59fe2be9458be